Update SciRep badge link in README (#104)
ivadomed/model-spinal-rootlets: README.md

# Automatic Segmentation of Spinal Nerve Rootlets 

[![ScientificReports](https://img.shields.io/badge/ScientificReports-s41598--026--49164--0-status.svg)](https://www.nature.com/articles/s41598-026-49164-0)
[![DOI](https://img.shields.io/badge/ImagingNeuroscience-10.1162/imag_a_00218-status.svg)](https://doi.org/10.1162/imag_a_00218)

![rootlets_model_v2](https://github.com/user-attachments/assets/74b297fe-f7d6-4990-b30a-89b231fad4a5)

This repository contains the code for deep learning-based segmentation of the spinal nerve rootlets. 
The code is based on the [nnUNet framework](https://github.com/MIC-DKFZ/nnUNet). The segmentation model is available as part of [Spinal Cord Toolbox (SCT)](https://spinalcordtoolbox.com) via the `sct_deepseg` function; see [How to use the model](#how-to-use-the-model) section for details.

## Citation Info

> If you find this work and/or code useful for your research, please cite the [RootletSeg paper](https://www.nature.com/articles/s41598-026-49164-0#article-info) (C2 - T1 dorsal and ventral rootlet segmentation):

Reviewer: I think we don't need to be so specific. 

```suggestion
If you find this work and/or code useful for your research, please cite the following papers:
```

```bibtex

> @article{Krejci202652,

Reviewer: To be consistent with the other ref

```suggestion
@article{10.1038/s41598-026-49164-0,
```

  author = {Kateřina Krejčí and Jiří Chmelík and Sandrine Bédard and Falk Eippert and Ulrike Horn and Virginie Callot and Julien Cohen-Adad and Jan Valošek},
  journal = {Scientific Reports},
  title = {Segmentation of spinal rootlets across MRI contrasts with RootletSeg},
  publisher = {Springer Science and Business Media},
  year = {2026-5-2},
  ISSN = {2045-2322},
  medium = {online},
  DOI = {10.1038/s41598-026-49164-0},
}
```

> or the [following paper](https://doi.org/10.1162/imag_a_00218) (C2 - C8 dorsal only rootlet segmentation):

Reviewer: ```suggestion

```
